packages/@jsii/go-runtime/build-tools/gen.ts

```diff
@@ -1,14 +1,35 @@
 #!/usr/bin/env npx ts-node
 
-import { copyFileSync, readdirSync } from 'fs';
+import * as fs from 'fs-extra';
 import { join, resolve } from 'path';
 
-const EMBEDDED_SOURCE = resolve(__dirname, '..', '..', 'runtime', 'webpack');
+const EMBEDDED_SOURCE = join(
+  resolve(__dirname, '..', '..', 'runtime', 'webpack'), 'jsii-runtime.js'
+);
 
-for (const filename of readdirSync(EMBEDDED_SOURCE)) {
-  const filepath = join(EMBEDDED_SOURCE, filename);
-  copyFileSync(
-    filepath,
-    resolve(__dirname, '..', 'jsii-experimental', 'embedded', filename),
-  );
+const RT_DIR = resolve(__dirname, '..', 'assets');
+const RT_PATH = resolve(RT_DIR, 'assets.go');
+
+fs.removeSync(RT_PATH);
+
+fs.open(RT_PATH, 'w', async (err, fd) => {
+  if (err) {
+    console.error(err);
+    return;
```

Comment:
Should this has thrown an error instead? In case of error, we'd appear to have succeeded (short of parsing STDERR).